tests/filters/test_background_color.py

```diff
@@ -16,18 +16,18 @@
 
 
 class BackgroundColorFilterTestCase(FilterTestCase):
+    def config_context(context):
```

Comment:
```suggestion
    def config_context(self, context):
```

Otherwise it'll fail.

Are you running testes locally?